store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -2630,6 +2630,7 @@ public static interface ProvisioningValidator {
         static final String DELETE_ACCOUNT = "deleteAccount";
         static final String CREATE_COS = "createCos";
         static final String DELETE_COS = "deleteCos";
+        static final String DELETE_COS_SUCCEEDED = "deleteCosSucceeded";
```

Comment:
Here we introduced new action deleteCosSucceeded.

Reasons

1. Suppose license is not valid, delete cos is getting failed from license-tools to prevent the deletion
2. This is happening after performing the cos deletion from mailbox end.
3. After reloading the page, we can see the cos is deleted
4. So to prevent the cos deletion from mailbox end, we are doing the license in deleteCos action
5. Releasings seats are handled in deleteCosSucceeded